source/type/pb_converter.cpp

```diff
@@ -1015,7 +1015,35 @@ void PbCreator::Filler::fill_pb_object(const nd::Impact* impact, pbnavitia::Impa
     auto pb_severity = pb_impact->mutable_severity();
     pb_severity->set_name(impact->severity->wording);
     pb_severity->set_color(impact->severity->color);
-    pb_severity->set_effect(to_string(impact->severity->effect));
+    switch (impact->severity->effect) {
```

Comment:
Would it be cleaner to separate the enum conversion into a separate function ? Like :
```c++
pbnavitia::Severity_Effect get_severity_effect(nd::Effect::Effect e) {
	switch(e) 
	{
	case nd::Effect::NO_SERVICE: 
		return pbnavitia::Severity_Effect::Severity_Effect_NO_SERVICE;
    case nd::Effect::REDUCED_SERVICE: 
		return pbnavitia::Severity_Effect::Severity_Effect_REDUCED_SERVICE;
    [...]
	}
}

[...]
pb_severity->set_effect(get_severity_effect(impact->severity->effect));
```